compiler/rustc_target/src/spec/mod.rs

```diff
@@ -1998,6 +1999,14 @@ impl TargetWarnings {
     }
 }
 
+/// For the [`Target::check_consistency`] function, determines whether the given target is a builtin or a JSON
+/// target.
```

Comment:
pls summarize that we check less things for json targets

Reply:
Added a doc comment to check_consistency.

Should we keep the FIXME or consider it resolved?

Reply:
I guess the question is whether we want other default args for json targets?

Reply:
Okay, I updated the comment accordingly.